rome_app/views.py

```diff
@@ -598,12 +600,14 @@ def essay_detail(request, essay_slug):
     related_list=[]
     for work in essay.related_works():
         current_work={}
-        current_work['title']=work['primary_title']
-        current_work['creator']=work.get('creator')
+        current_work['title']=work['primary_title'].encode('ascii', 'ignore').decode('ascii')
```

Comment:
These lines where you're encoding the data and then decoding it again - I don't think you need to do that. In related_works(), the data comes through r.json(). I just looked at that, and it looks like that data is unicode already. So you should be able to just use work['primary_title'], and leave the rest of the line off.

If you had an issue somewhere, though, where that didn't work, let me know and we can try to address it.